src/Tribe/Views/V2/Hooks.php

```diff
@@ -115,6 +115,9 @@ protected function add_filters() {
 
 		add_filter( 'tribe_customizer_inline_stylesheets', [ $this, 'customizer_inline_stylesheets' ], 12, 2 );
 		add_filter( 'tribe_events_views_v2_view_data', [ View_Utils::class, 'clean_data' ] );
+
+		// Customizer
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 21dba030ad45af1616bf4e6792d52ed6</sub>